packages/react-app/src/App.jsx

```diff
@@ -77,6 +71,7 @@ function App(props) {
 
   const [injectedProvider, setInjectedProvider] = useState();
   const [address, setAddress] = useState();
+  const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
What is the use for this?

I see you are doing `isLoaded ? POLL_TIME : 500,` below, but not sure what is the intention with it.

Reply:
ya, i think it's unnecessary I added it while building the logic. 
the intention was to add 500ms of time at the start to load price or balance etc quickly